tests/network/checkup_framework/utils.py

```diff
@@ -174,6 +174,7 @@ def compose_configmap_data(
     data_dict = {
         "spec.timeout": timeout,
         "spec.param.sampleDurationSeconds": sample_duration_seconds,
+        "spec.param.verbose": "true",
```

Comment:
@Anatw based on @orelmisan 's comment https://github.com/RedHatQE/openshift-virtualization-tests/pull/85#issuecomment-2616134940, compose_configmap_data() is not the right place for this change. If DPDK checkup only respects `verbose` option, then only calls to  compose_configmap_data() from dpdk fixtures should be setting this option.

Reply:
I'm sad now.

Reply:
Ack. This PR is now based on Edi's PR. Only the last commit should be reviewed.